backend/app/services/codegraph/repo_service.py

```
@@ -0,0 +1,285 @@
+                            "indexing_status": "failed",
+                            "last_error": error_msg
+                        }).eq("repository_full_name", repo_info['full_name']).eq(
+                            "is_deleted", False
+                        ).execute()
+
+                        return {"status": "error", "message": f"Indexing failed: {error_msg}"}
+
+        except ValueError as e:
+            return {"status": "error", "message": str(e)}
+        except aiohttp.ClientError as e:
+            logger.error(f"Network error indexing {repo_input}: {e}")
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)